snmp/datadog_checks/snmp/data/profiles/cisco-catalyst.yaml

```diff
@@ -10,6 +10,9 @@ device:
 
 sysobjectid:
   - 1.3.6.1.4.1.9.1.111
+  - 1.3.6.1.4.1.9.1.150
+  - 1.3.6.1.4.1.9.1.151
+  - 1.3.6.1.4.1.9.1.152
```

Comment:
Could you also add the model/oidName as comment to each `sysobjectid` ? I know the existing sysobjectid doesn't have that, but having that can help readability.